contrib/rocksdb/template.py

```diff
@@ -0,0 +1,58 @@
+pkgname = "rocksdb"
+pkgver = "9.4.0"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_target = "static_lib"
+hostmakedepends = [
+    "bash",
+    "gmake",
+    "perl",
+    "pkgconf",
+]
+makedepends = [
+    "bzip2-devel",
+    "gflags-devel-static",
+    "linux-headers",
+    "lz4-devel",
+    "snappy-devel",
+    "zstd-devel",
+    "zlib-ng-compat-devel",
+]
+pkgdesc = "Embeddable, persistent key-value store for fast storage"
+maintainer = "Gnarwhal <git.aspect893@passmail.net>"
+license = "GPL-2.0-or-later OR Apache-2.0 OR custom:leveldb"
+url = "https://rocksdb.org"
+source = f"https://github.com/facebook/rocksdb/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "1f829976aa24b8ba432e156f52c9e0f0bd89c46dc0cc5a9a628ea70571c1551c"
+hardening = ["!pie"]
```

Comment:
not sure what this was there for since it doesn't affect anything, but for the shared build to work you'll need

```suggestion
tool_flags = {"CXXFLAGS": ["-fPIC"]}
```